### PR TITLE
Bugfix/android playstore crash

### PR DESCRIPTION
**Pull request by fpohtmeh-github**

### CODE REVIEW

File: src/controllers/CloudFilesController.cpp [hunk 1]
```diff
@@ -105,7 +105,9 @@ CloudFilesModel *Self::model()
 
 void Self::switchToRootFolder()
 {
-    switchToHierarchy({ rootFolder() });
+    if (FileUtils::forceCreateDir(rootFolder()->localPath())) {
```
Comment (SergeySeroshtan) [inline src/controllers/CloudFilesController.cpp]: This line will will case `abort` if folder will not be created

**Review by SergeySeroshtan — CHANGES_REQUESTED** [review]

File: src/controllers/CloudFilesController.cpp (hunk 1 above)
Comment (fpohtmeh-github) [reply src/controllers/CloudFilesController.cpp]: This code is outdated now

File: src/logging/VSQLogWorker.cpp [hunk 2]
```diff
@@ -51,9 +51,9 @@ VSQLogWorker::VSQLogWorker(QObject *parent) : QObject(parent), m_logFile(), m_lo
 
 void VSQLogWorker::processMessage(QtMsgType type, const VSQMessageLogContext &context, const QString &message)
 {
-#ifdef QT_DEBUG
+    //#ifdef QT_DEBUG
```
Comment (SergeySeroshtan) [inline src/logging/VSQLogWorker.cpp]: This changes should be reverted.

File: src/logging/VSQLogWorker.cpp [hunk 3]
```diff
@@ -51,9 +51,9 @@ VSQLogWorker::VSQLogWorker(QObject *parent) : QObject(parent), m_logFile(), m_lo
...
     consoleMessageHandler(type, context, message);
-#endif
+    //#endif
```
Comment (SergeySeroshtan) [inline src/logging/VSQLogWorker.cpp]: This changes should be reverted.

File: CMakeLists.txt [hunk 4]
```diff
@@ -947,7 +947,7 @@ target_compile_definitions(${VS_TARGET_NAME}
         $<BUILD_INTERFACE:VS_MOBILE=$<IN_LIST:${VS_PLATFORM},${VS_MOBILE_PLATFORM_LIST}>>
         $<BUILD_INTERFACE:VS_PUSHNOTIFICATIONS=$<IN_LIST:${VS_PLATFORM},${VS_MOBILE_PLATFORM_LIST}>>
 
-        $<BUILD_INTERFACE:$<$<CONFIG:Release>:QT_NO_DEBUG_OUTPUT>>
+#        $<BUILD_INTERFACE:$<$<CONFIG:Release>:QT_NO_DEBUG_OUTPUT>>
```
Comment (SergeySeroshtan) [inline CMakeLists.txt]: This changes should be reverted.

**Review by SergeySeroshtan — CHANGES_REQUESTED** [review]
And make a fancy name for the PR )

**Review by SergeySeroshtan — APPROVED** [review]

**Pull request merged**